packages/flutter/test/widgets/table_test.dart

```diff
@@ -930,29 +930,14 @@ void main() {
     },
   );
 
-  testWidgets(
-    'Table widget - Default textBaseline is set to TableBaseline.alphabetic',
-    (WidgetTester tester) async {
-      await tester.pumpWidget(
-        Directionality(
-          textDirection: TextDirection.ltr,
-          child: Table(
-            defaultVerticalAlignment: TableCellVerticalAlignment.baseline,
-            children: const <TableRow>[
-              TableRow(
-                children: <Widget>[
-                  Text('Some Text'),
-                ],
-              ),
-            ],
-          ),
-        ),
-      );
-
-      final RenderTable table = tester.renderObject(find.byType(Table));
-      expect(table.textBaseline, TextBaseline.alphabetic);
-    },
-  );
+  testWidgets('Table widget - Default textBaseline is null', (WidgetTester tester) async {
+    try {
+      Table(defaultVerticalAlignment: TableCellVerticalAlignment.baseline);
+      throw TestFailure('Table should throw when missing necessary baseline information.');
+    } on AssertionError catch (error) {
```

Comment:
instead of throwing in the test and catching, could we make use of `expect` and `throwsA` methods are mentioned [here](https://github.com/dart-lang/sdk/issues/39305) and [here](https://github.com/dart-lang/sdk/issues/34530)

Reply:
I couldn't figure out how to make throwsA verify the text of the exception. Can you elaborate on what you'd like it to look like?

Reply:
You can use `.having` for that:

https://github.com/flutter/flutter/blob/2783f8e2e14efec8b7e08f668dde61c40d128c24/packages/flutter/test/foundation/diagnostics_test.dart#L903-L910

Reply:
Thanks!

(this is why my ideal test framework has no `expect` and no matchers. Just use regular Dart. No need to learn crazy APIs when regular code can do the same thing in fewer lines of code, no shame in not knowing the random test library APIs. :-P)